debug.xml

```diff
@@ -41,10 +41,11 @@
     <entry key='logger.level'>all</entry>
     <entry key='logger.file'>target/tracker-server.log</entry>
 
+    <entry key='deviceManager.lookupGroupsAttribute'>true</entry>
+
     <entry key='event.enable'>true</entry>
-    <entry key='event.suppressRepeated'>60</entry>
     <entry key='event.overspeedHandler'>true</entry>
-    <entry key='event.globalSpeedLimit'>90</entry>
+    <entry key='event.overspeed.riseOnce'>true</entry>
```

Comment:
I guess we can. Default doesn't really matter that much because you can
always explicitly specify what you want.